### ISSUE TITLE
Can only generate one path. Every path after generates ClassNotFoundException: MatchInfoData

### ISSUE THREAD

**Issue by Gregadeaux** [wpilibsuite/PathWeaver#165]
**Describe the bug**  
On a fresh build, I was able to create a path, and then export it to json. But every other path I tried to make (or when I tried to update the original path), I received a ClassNotFoundException and it was not able to export the path.

![image](https://user-images.githubusercontent.com/472886/71752098-69be7f00-2e43-11ea-8975-d3fc7fe65468.png)

Running on Windows.

**Comment by carbotaniuman** [comment]
I cannot reproduce this. Can you send your project folder?

**Comment by agasser** [comment]
I was getting this last night when running PathWeaver on `master` with `gradle run`. I think it had to do with an incompatible version of WPILib, or maybe a bad WPILib build snapshot.

I worked around it by changing the WPILib version range to 2019.1.2 in `build.gradle.kts`.

```
    compile(group = "edu.wpi.first.wpiutil", name = "wpiutil-java", version = "2020.1.2")
    compile(group = "edu.wpi.first.wpilibj", name = "wpilibj-java", version = "2020.1.2")
```

**Comment by carbotaniuman** [comment]
Ughhh. I know what's going on now. If you try and generate an invalid path it breaks. I'll push an update today or tommorow.

**Comment by agasser** [comment]
Yeah, my workaround above was a fluke.

**Comment by jasondaming** [comment]
@carbotaniuman can you look at this again or more fully explain what is going on?

**Comment by jasondaming** [comment]
Closing as fixed via #169
